WeakAuras/GenericTrigger.lua

```diff
@@ -3135,7 +3135,7 @@ function GenericTrigger.GetOverlayInfo(data, triggernum)
     if (trigger.custom_type == "stateupdate") then
       local count = 0;
       local variables = events[data.id][triggernum].tsuConditionVariables;
-      if (variables) then
+      if (variables and type(variables) == "table") then
```

Comment:
```suggestion
      if (type(variables) == "table") then
```
if variables is a table, then it will also be truthy